### DISCUSSION TITLE
p167 SECTION-065 リスト17.3について

### DISCUSSION THREAD

**Discussion by Mo3g4u** [budougumi0617/go_todo_app#25] [Ideas]
間違っていたらすみません。
サンプル通り書いたらエラーでした。

store/store.goのTaskStore構造体に合わせる場合は以下でしょうか？

- １つ目
```go
Tasks = &TaskStore{Tasks: map[int]*entity.Task{}}
```
↓
```go
Tasks = &TaskStore{Tasks: map[entity.TaskID]*entity.Task{}}
```

- 2つ目
```go
func (ts *TaskStore) Add(t *entity.Task) (int, error) {
```
↓
```go
func (ts *TaskStore) Add(t *entity.Task) (entity.TaskID, error) {
```

**Comment by budougumi0617** [comment]
ご指摘ありがとうございます。Twitterでも同様のご指摘いただいております。
おっしゃるとおり、defined typeを使うようにしたコードとそうでないコードが混じっている状態です。
サンプルコードと誌面正誤表の修正をさせていただきます。

**Comment by budougumi0617** [comment]
ご指摘を正誤表とサンプルコードに反映しました。
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.2

**Reply by Mo3g4u** [reply]
ご対応ありがとうございます！
